.github/workflows/version-increments.yml

```diff
@@ -0,0 +1,14 @@
+name: Publish Version Check Results
+
+on:
+  workflow_run:
+    workflows: [ 'Pull-Request Checks' ]
```

Comment:
So it does  wait for all jobs and not only for "check-versions"`?

Reply:
Exactly.
As far as I can tell only the completion of an entire workflow triggers an event on which another workflow can run:
- https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_run

If you now a way to react on the completion of a single job in a new workflow that can run with access to secrets, please let me know.
But all other jobs should run much faster, so it's not really a problem atm.